frontend/src/js/controllers/challengeHostTeamsCtrl.js

```diff
@@ -369,6 +369,82 @@
             }, function() {});
         };
 
+// Delete a particular host
+vm.confirmMemberDelete = function(ev, hostTeamId, hostId) {
+    ev.stopPropagation();
+    // Appending dialog to document.body to cover sidenav in docs app
+    var confirm = $mdDialog.confirm()
+        .title('Would you like to remove this member?')
+        .textContent('Note: This action will remove this member from the team.')
+        .ariaLabel('Lucky day')
+        .targetEvent(ev)
+        .ok('Yes')
+        .cancel("No");
+
+    $mdDialog.show(confirm).then(function() {
+        vm.startLoader();
+        var parameters = {};
+        parameters.url = 'hosts/challenge_host_team/' + hostTeamId + '/challenge_host/' + hostId;
+        parameters.method = 'DELETE';
+        parameters.data = {};
+        parameters.token = userKey;
+        parameters.callback = {
+            onSuccess: function() {
+                vm.team.error = false;
+                $rootScope.notify("info", "You have removed the member successfully");
```

Comment:
change this `You have removed the member successfully` to `Member removed successfully`